pkg/oidcclient/login.go

```diff
@@ -343,6 +346,7 @@ func (h *handlerState) initOIDCDiscovery() error {
 }
 
 func (h *handlerState) tokenExchangeRFC8693(baseToken *oidctypes.Token) (*oidctypes.Token, error) {
+	plog.Debug("Pinniped: Performing RFC8693 token exchange", "requested audience", h.requestedAudience)
```

Comment:
```suggestion
	plog.Debug("Pinniped: Performing RFC8693 token exchange", "requestedAudience", h.requestedAudience)
```

Another small style nit but Kubernetes convention is "lowerCamelCase" for structured logging keys.